### PR TITLE
Center aligning the submit button for the add and edit page - (Solves - #143)

### PR DESCRIPTION
**Pull request by pranav1698**
### Description :
Aligning the submit button at the center of the page on each Add and Edit page like AddAdapter, EditEnforcer etc
Similar to #142 
It includes alignment of code issues as recommended by prettier js

### Screenshot of changes :

Screenshot from 2021-01-06 23-38-35
Screenshot from 2021-01-16 01-05-56
Screenshot from 2021-01-16 01-06-02

### Please ensure Checkmark the following :

* [x] - Tested Localy
* [x] - Checked in IE11 and Firefox
* [ ] - Checked in mobile
* [ ] - Backend work
* [x] - Frontend work
* [x] - Bug Fixes
* [ ] - Enhancements

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Xiaoyang-Gu plz review.

File: web/src/pages/dashboard/AdapterPage/EditAdapter.js [hunk 1]
```diff
@@ -1,157 +1,148 @@
...
+        xs: {
+          span: 24,
+          offset: 0,
+        },
+        sm: {
+          span: 16,
+          offset: 8,
+        },
+      },
+    };
```
Comment (Xiaoyang-Gu) [inline web/src/pages/dashboard/AdapterPage/EditAdapter.js:44]: I recommend to use `{...tailFormItemLayout}` to fix the Button layout of the FormItem.
But if you decide to use your method, please delete these lines.

Comment (pranav1698) [reply web/src/pages/dashboard/AdapterPage/EditAdapter.js:44]: I think this method will help in realigning the button easily, if we want to change the style next time
I will do the required changes

**Comment by pranav1698** [comment]
@hsluoyz please review

File: web/src/pages/dashboard/AdapterPage/EditAdapter.js (hunk 1 above)
Comment (hsluoyz) [reply web/src/pages/dashboard/AdapterPage/EditAdapter.js]: @Xiaoyang-Gu

**Review by Xiaoyang-Gu — APPROVED** [review]

**Review by Xiaoyang-Gu — APPROVED** [review]

**Pull request merged**